### PR TITLE
Add test to HandLandMarkViewModel to upgrade the coverage.

### PR DESCRIPTION
**Pull request by SaFouNaldo**
- Formatting files & creating the tests using mockito 
- Remoove duplicates in the build.gradle.kts 
image
Future work : Add tests for HandLandMarkImplementation

### CODE REVIEW

File: app/build.gradle.kts
Comment (ziyadbagh) [inline app/build.gradle.kts:1]: Nice cleanup on the redundant code!

File: app/src/test/java/com/github/se/signify/model/hand/HandLandMarkViewModelTest.kt
Comment (ziyadbagh) [inline app/src/test/java/com/github/se/signify/model/hand/HandLandMarkViewModelTest.kt:1]: Solid work on the tests, great coverage!

**Review by ziyadbagh — COMMENTED** [review]
Everything looks good to me, I will approve this pull request

**Review by ziyadbagh — APPROVED** [review]

**Review by ThomasMPont — APPROVED** [review]
Small and efficient changes, Sonarcloud has nothing to say. Looks good to merge.

**Pull request merged**